src/common/utils/netlify.ts

```diff
@@ -24,3 +28,22 @@ export function getPageContent(
 
 export type { DonatePageContent };
 export { donateContent };
+
+// TODO (Chelsi): clean up if keeping?
+export function getDataDisclaimers(
+  fips: string,
+  stateCode: string,
+  countyId?: string,
+): Disclaimer[] {
+  const disclaimersContent = dataDisclaimers.disclaimers;
+  if (countyId) {
+    return disclaimersContent.filter(
+      disclaimer =>
+        disclaimer.stateCodes?.split(',').includes(stateCode) ||
+        disclaimer.fipsCodes?.split(',').includes(fips),
+    );
```

Comment:
I think we need to filter out invalid fips codes here (checking whether or not they are in the state or county fips list)